src/main/java/org/example/MainServlet.java

```diff
@@ -24,7 +24,7 @@ public class MainServlet extends HttpServlet {
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
 
-        LOGGER.info("We enter on path {}", req);
+        LOGGER.info("We enter on path {}", req.getRequestURI());
```

Comment:
Add string varieble before you add a 'req' to logger.info. Make string from httpServletRequest.

Reply:
fixed